lib/sidekiq/ultimate/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module Sidekiq
   module Ultimate
-    VERSION = "2.0.0"
+    VERSION = "2.0.1"
```

Comment:
Imo - this should be not patch, but at least minor version change

Reply:
That's true.